### ISSUE TITLE
bot doesn't play sounds

### ISSUE THREAD

**Issue by aQuAiS18** [ks00908/doot-doot#36]
bot didn't play sound and won't play other sounds, saying that it has to finish playing the sound that it didn't play. tried kicking bot and adding back again but that didn't work.

**Comment by ks00908** [comment]
Hi

Thank you for reaching out!
I saw that happen on few diffrent servers, and i'm curently working on tracking down the issue, however so far i was unable to find any specific reproduction steps.

If you could add what you were doing before it happened and ServerID of server where it is happened it would likely help me to try piece together what causes it.

Thank you
Krzysztof Szypuła


---
Helpful Links:
[How to get ServerID](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-)

**Comment by aQuAiS18** [comment]
sever ID 827078735995076619
me and my friend were just messing around, playing all the sounds until it didn't play the Godno one again. that's when it happened. btw I tried using it again and it somehow works now.

**Comment by ks00908** [comment]
Thank you for providing the ID and description of how it happened!

I will start looking through logs again when i  get back home from work, hopefully i will finnally find somethig that gives me a clue what makes code think its playing sound when it isnt.

as to it starting working: When i saw your issue i restarted bot, it at least temporarly fixes the issue. I might honestly start restarting it hourly just to minimise impact, as much as id prefer to avoid that

**Comment by aQuAiS18** [comment]
Np. Thanks 🙂

**Comment by ks00908** [comment]
Hi!
so just to give some update to report:

1. I checked logs, and as ussual there were no errors reported by bot code
2. I decided to investigate host itself and installed additional monitoring software
3. Thanks to additional software, i discovered that i ocassionally use all avaible memory, however i wasnt able to confirm if that causes any guild to bug out.

My curent running theory is that resource usage being too high causes code to belive it is still playing sound even after it finishes, i plan to upgrade my server in near future which if i am right about resource usage being the problem should solve it
